alarm/utils.py

```diff
@@ -0,0 +1,122 @@
+from alarm.models import Coin, Candle
+import websocket
+import json
+import ssl
+
+
+def connect_binance_socket(currencies, intervals):
```

Comment:
Ok, we have everything encapsulated in a single method. 
One step further is to extract a method, which connects to a single socket.
But we will not do that for now, as it generally looks ok.

Reply:
++